internals/webpack/webpack.base.babel.js

```diff
@@ -4,6 +4,30 @@
 
 const path = require('path');
 const webpack = require('webpack');
+const dotenv = require('dotenv').config(); // eslint-disable-line no-unused-vars
```

Comment:
the config method is the one responsible of loading the vars into the `process.env` object.